src/Spryker/Zed/RabbitMq/Communication/Plugin/Queue/RabbitMqQueueMetricsExpanderPlugin.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace Spryker\Zed\RabbitMq\Communication\Plugin\Queue;
+
+use Generated\Shared\Transfer\QueueMetricsRequestTransfer;
+use Generated\Shared\Transfer\QueueMetricsResponseTransfer;
+use Spryker\Client\RabbitMq\Model\RabbitMqAdapter;
+use Spryker\Zed\Kernel\Communication\AbstractPlugin;
+use Spryker\Zed\QueueExtension\Dependency\Plugin\QueueMetricsExpanderPluginInterface;
+
+/**
+ * {@inheritDoc}
+ *
+ * @api
```

Comment:
If nothing was changed, we are not adding api tag to the class itself. But we add plugin description here

Reply:
Code style requires api tag